lms/templates/courseware_search/search_list.underscore

```diff
@@ -0,0 +1,24 @@
+<div class="search-info">
+    <%- gettext("Searching") %> <%= courseName %>
+    <div class="search-count">
+        <span class="search-count-total"><%- totalCount %></span>  <%- ngettext("result", "results", totalCount) %>
```

Comment:
Not all languages will have the number before the word "results". You should do something like this:
```js
<%
  var fmt = ngettext(
    "{total} result",
    "{total} results",
    totalCount
  );
  var totalSpan = '<span class="search-count-total">' + totalCount + '</span>';
  print(fmt.replace("{total}", totalSpan);
%>
```
That way, the translators can move around the `{total}` token in the phrase as appropriate for their language.